### PR TITLE
Test Component PostModal #157

### PR DESCRIPTION
**Pull request by Khaalid-oh**
## Test Component PostModal #157

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by Vayras** [comment]
@Khaalid-oh jest is failing did you test this one locally?

**Comment by kevkevinpal** [comment]
looks like nock is failing during the test

I'm seeing the following error
```
  ●  Cannot log after tests are done. Did you forget to wait for something async in your test?
    Attempted to log "Error getUserDropdownOrganizations FetchError {
      message: 'request to http://localhost:5002/organizations/user/dropdown/1 failed, reason: Nock: No match for request {\n' +
        '  "method": "GET",\n' +
        '  "url": "http://localhost:5002/organizations/user/dropdown/1",\n' +
        '  "headers": {\n' +
        '    "content-type": [\n' +
        '      "application/json"\n' +
        '    ],\n' +
        '    "accept": [\n' +
        '      "*/*"\n' +
        '    ],\n' +
        '    "user-agent": [\n' +
        '      "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"\n' +
        '    ],\n' +
        '    "accept-encoding": [\n' +
        '      "gzip,deflate"\n' +
        '    ]\n' +
        '  }\n' +
        '}',
      type: 'system',
      errno: 'ERR_NOCK_NO_MATCH',
      code: 'ERR_NOCK_NO_MATCH'
    }".



      at console.log (node_modules/@jest/console/build/BufferedConsole.js:199:10)
      at MainStore.<anonymous> (src/store/main.ts:53339:21)
      at step (src/store/main.ts:46511:19)
      at Object.throw (src/store/main.ts:46277:14)
      at rejected (src/store/main.ts:46188:32)


  ●  Cannot log after tests are done. Did you forget to wait for something async in your test?
    Attempted to log "FetchError {
      message: 'request to http://localhost:5002/people?page=1&search=&sortBy=last_login&limit=100 failed, reason: Nock: No match for request {\n' +
        '  "method": "GET",\n' +
        '  "url": "http://localhost:5002/people?page=1&search=&sortBy=last_login&limit=100",\n' +
        '  "headers": {\n' +
        '    "accept": [\n' +
        '      "*/*"\n' +
        '    ],\n' +
        '    "user-agent": [\n' +
        '      "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"\n' +
        '    ],\n' +
        '    "accept-encoding": [\n' +
        '      "gzip,deflate"\n' +
        '    ]\n' +
        '  }\n' +
        '}',
      type: 'system',
      errno: 'ERR_NOCK_NO_MATCH',
      code: 'ERR_NOCK_NO_MATCH'
    }".
```

**Comment by Vayras** [comment]
@Khaalid-oh  ` nock(user.url).get('/person/id/1').reply(200, user);`  add a url with an Id that exist maybe that will solve the nock error

**Comment by Khaalid-oh** [comment]
> nock(user.url).get('/person/id/1').reply(200, user);
Yes, I did. All test passed when I test it locally 

Screenshot 2024-02-14 at 18 33 41

**Comment by Khaalid-oh** [comment]
> @Khaalid-oh ` nock(user.url).get('/person/id/1').reply(200, user);` add a url with an Id that exist maybe that will solve the nock error

Thanks, I've tried that

**Comment by Khaalid-oh** [comment]
> looks like nock is failing during the test
> 
> I'm seeing the following error
> 
> ```
>   ●  Cannot log after tests are done. Did you forget to wait for something async in your test?
>     Attempted to log "Error getUserDropdownOrganizations FetchError {
>       message: 'request to http://localhost:5002/organizations/user/dropdown/1 failed, reason: Nock: No match for request {\n' +
>         '  "method": "GET",\n' +
>         '  "url": "http://localhost:5002/organizations/user/dropdown/1",\n' +
>         '  "headers": {\n' +
>         '    "content-type": [\n' +
>         '      "application/json"\n' +
>         '    ],\n' +
>         '    "accept": [\n' +
>         '      "*/*"\n' +
>         '    ],\n' +
>         '    "user-agent": [\n' +
>         '      "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"\n' +
>         '    ],\n' +
>         '    "accept-encoding": [\n' +
>         '      "gzip,deflate"\n' +
>         '    ]\n' +
>         '  }\n' +
>         '}',
>       type: 'system',
>       errno: 'ERR_NOCK_NO_MATCH',
>       code: 'ERR_NOCK_NO_MATCH'
>     }".
> 
> 
> 
>       at console.log (node_modules/@jest/console/build/BufferedConsole.js:199:10)
>       at MainStore.<anonymous> (src/store/main.ts:53339:21)
>       at step (src/store/main.ts:46511:19)
>       at Object.throw (src/store/main.ts:46277:14)
>       at rejected (src/store/main.ts:46188:32)
> 
> 
>   ●  Cannot log after tests are done. Did you forget to wait for something async in your test?
>     Attempted to log "FetchError {
>       message: 'request to http://localhost:5002/people?page=1&search=&sortBy=last_login&limit=100 failed, reason: Nock: No match for request {\n' +
>         '  "method": "GET",\n' +
>         '  "url": "http://localhost:5002/people?page=1&search=&sortBy=last_login&limit=100",\n' +
>         '  "headers": {\n' +
>         '    "accept": [\n' +
>         '      "*/*"\n' +
>         '    ],\n' +
>         '    "user-agent": [\n' +
>         '      "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"\n' +
>         '    ],\n' +
>         '    "accept-encoding": [\n' +
>         '      "gzip,deflate"\n' +
>         '    ]\n' +
>         '  }\n' +
>         '}',
>       type: 'system',
>       errno: 'ERR_NOCK_NO_MATCH',
>       code: 'ERR_NOCK_NO_MATCH'
>     }".
> ```

Yeah, I'm currently looking into why it's failing now

**Comment by elraphty** [comment]
@Khaalid-oh The tests are failing

**Comment by Khaalid-oh** [comment]
> @Khaalid-oh The tests are failing
I'm currently on it... I'm getting this FetchError: request to http://localhost:5002/person/id/1 failed, reason: Nock: Disallowed net connect for "localhost:5002/person/id/1" now

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (elraphty) [inline .idea/inspectionProfiles/Project_Default.xml:1]: @Khaalid-oh remove all these .idea IntelliJ files

Comment (Khaalid-oh) [reply .idea/inspectionProfiles/Project_Default.xml]: Done

**Pull request merged**